### ISSUE TITLE
Fine tune model

### ISSUE THREAD

**Issue by alexrobincrabbe** [alexrobincrabbe/Brain_tumour_diagnostic#21]
As a developer I can fine tune the model hyperparameters so that the model meets the required minimum metrics on the text set

### Acceptance Criteria

- A precision and recall of at least 0.65  is achieved accross all labals

### Tasks

- [x] Tune parameters
- [x] Train model
- [x] Evaluate on test set
